lib/id3c/cli/command/etl/__init__.py

```diff
@@ -419,6 +420,128 @@ def upsert_encounter_location(db: DatabaseSession,
         assert cursor.rowcount == 1, "Upsert affected no rows!"
 
 
+def upsert_presence_absence(db: DatabaseSession,
+                            identifier: str,
+                            sample_id: int,
+                            target_id: int,
+                            present: bool,
+                            details: dict) -> Any:
+    """
+    Upsert presence_absence by its *identifier*.
+
+    Confirmed with Samplify that their numeric identifier for each test is stable
+    and persistent.
+    """
+    LOG.debug(f"Upserting presence_absence «{identifier}»")
+
+    data = {
+        "identifier": f"NWGC_{identifier}",
```

Comment:
Just wanted to flag that in the most recent changes to `etl presence-absence`, this got updated to 
```
"identifier": identifier,
```